onnx/version_converter/convert.h

```diff
@@ -545,6 +545,7 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(make_unique<CompatibleAdapter>("CastLike", OpSetID(18), OpSetID(19)));
     registerAdapter(make_unique<CompatibleAdapter>("Constant", OpSetID(18), OpSetID(19)));
     registerAdapter(make_unique<CompatibleAdapter>("DequantizeLinear", OpSetID(18), OpSetID(19)));
+    registerAdapter(make_unique<CompatibleAdapter>("GridSample", OpSetID(18), OpSetID(19)));
```

Comment:
The GridSample Opset 19 is not backward compatible as the modes become `nearest`, `linear`, `cubic` instead of `nearest`, `bilinear`, `bicubic`. Need to create a new adapater I think.

Reply:
Added the adapter.